examples/adapt/object_detection/README.md

```diff
@@ -0,0 +1,212 @@
+
+$ docker run -it -d --gpus all -v /dev/shm:/dev/shm -v $(pwd):/workspace --name nyuntam_adapt_docker --network=host nyunadmin/nyunzero_adapt:v0.1 bash 
+
+$ docker exec -it nyuntam_adapt_docker /bin/bash
+
+$ cd /workspace/nyuntam
+```
+
+## Dataset
+
+For this experiment we will be using the [face-det](https://universe.roboflow.com/mohamed-traore-2ekkp/face-detection-mik1i) dataset from roboflow. The dataset is downloaded in coco format. The data folder structure is formatted in the folowing way: 
+
+```bash
+face-det
+└── train
+|   └── annotations_coco.json
+|   └── image1.jpg
+|   └── image2.jpg
+    ...
+└── validation
+    └── annotations_coco.json
+    └── image1.jpg
+    └── image2.jpg
+
+```
+
+## Configuration
+
+The following YAML file is used for setting up the experiment : 
+
+```yaml
+#DATASET_ARGS :
+JOB_SERVICE: Adapt
+TRAIN_DIR : 'train/'
+VAL_DIR : 'validation/'
+TEST_DIR : 
+CUSTOM_DATASET_PATH : "/workspace/nyuntam/face_det"
+
+#MODEL_ARGS :
+MODEL : 'rtmdet' 
+MODEL_PATH :  'rtmdet_tiny_8xb32-300e_coco'  # config of model in case of mmdet
+CACHE_BOOL : False
+LOCAL_MODEL_PATH:  #This is empty becasue we are using the pretrained model from the internet. 
+
+#TRAINING_ARGS :
+SEED : 56
+DO_TRAIN : True
+DO_EVAL : True
+NUM_WORKERS : 4
+BATCH_SIZE : 1
+EPOCHS : 1
+OPTIMIZER : 'SGD' 
+LR : 5e-3 
+SCHEDULER_TYPE : 'CosineAnnealingLR'
+WEIGHT_DECAY : 0.0
+INTERVAL : 'steps'
+INTERVAL_STEPS : 4
+NO_OF_CHECKPOINTS : 5
+FP16 : False
+GRADIENT_ACCUMULATION_STEPS : 4
+GRADIENT_CHECKPOINTING : False
+BEGIN: 0
+END: 50
+
+#MMDET SPECIFIC ARGUMENTS :
+amp :  False
+resume  : False # ['auto']
+auto_scale_lr :  False
+cfg_options  : None
+launcher :  pytorch
+dest_root : './.mmdet_cache'       
+train_ann_file : 'annotations.coco.json'     
+val_ann_file : 'annotations.coco.json'        
+work_dir : ./results/mmdet  # same as output dir
+checkpoint_interval : 5
+
+#FINE_TUNING_ARGS :
+LAST_LAYER_TUNING : True
+FULL_FINE_TUNING :  False
+
+PEFT_METHOD : "LoRA"
+
+#LoRA_CONFIG :
+r : 32
+alpha : 16
+dropout : 0.1
+peft_type : 'LoRA'
+fan_in_fan_out : False
+init_lora_weights : True  
+
+TASK : 'object_detection'
+Library :  'MMDET' 
+cuda_id : '0,1'
+OUTPUT_DIR : "user_data/jobs/Aadpt/OBJDET"
+LOGGING_PATH : "user_data/logs/Adapt/OBJDET"
+MERGE_ADAPTER: True
+auto_select_modules: True
+SAVE_METHOD : 'state_dict'
+
+#DDP ARGS
+DDP: True
+num_nodes: 1
+```
+## Adapting the model
+With the yaml file configured, the adaptation process is initiated with the following command : 
+
+```bash 
+$ python run_dist.py --yaml_path examples/adapt/object_detection/config.yaml
+```
+
+Once the job starts, you will find the following directory structure in the `user_data` folder:
+
+```bash
+user_data/
+├── jobs
+│   └── Adapt
+│       └── OBJDET
+├── logs
+    └── Adapt
+        └── OBJDET
+            └── log.log
+
+```
+
+LoRa decreases the toal trainable parameters of the model by freezing the original model weights and updating LoRA adapters only. 
```

Comment:
nits -
spelling mistake "total".
Lets do a sanity check on spelling mistakes and grammars across the examples.